src/status_im/contexts/keycard/events.cljs

```diff
@@ -55,3 +56,19 @@
  (fn [{:keys [db]} [{:keys [on-cancel-event-vector]}]]
    (log/debug "[keycard] nfc started success")
    {:db (assoc-in db [:keycard :on-nfc-cancelled-event-vector] on-cancel-event-vector)}))
+
+(rf/reg-event-fx :keycard/on-action-with-pin-error
+ (fn [{:keys [db]} [error]]
+   (log/debug "[keycard] get keys error: " error)
+   (let [tag-was-lost?     (keycard.utils/tag-lost? (:error error))
+         pin-retries-count (keycard.utils/pin-retries (:error error))]
+     (if tag-was-lost?
+       {:db (assoc-in db [:keycard :pin :status] nil)}
+       (if (nil? pin-retries-count)
+         {:effects.utils/show-popup {:title "wrong-keycard"}}
```

Comment:
Should we move them to translations file?

Reply:
not really because it's temporary

Reply:
I don't know why these magic words don't work in my case. Maybe I am muggle 😅

https://github.com/status-im/status-mobile/pull/19968#discussion_r1598788990